extensions/admin-tool-ext-win/package.json

```diff
@@ -50,17 +50,17 @@
       "objectExplorer/item/context": [
         {
           "command": "adminToolExtWin.launchSsmsMinGswDialog",
-          "when": "isWindows && connectionProvider == MSSQL && nodeType && nodeType == Database",
+          "when": "isWindows && connectionProvider == MSSQL && nodeType && nodeType == Database !mssql:issqlondemand",
           "group": "z-AdminToolExt@1"
         },
```

Comment:
Missing && in this file

Reply:
You're missing the &&'s in a lot of other places too - make sure you go through and fix those